quote-server/server.js

```diff
@@ -3,18 +3,53 @@
 
 //load the 'express' module which makes writing webservers easy
 import express from "express";
+import fetch from "node-fetch";
 //load the quotes JSON
-import quotes from "./quotes.json" assert { type: "json" };
+// import quotes from "./quotes.json" assert { type: "json" };
+let quotes = [];
 
+const fetchQuotes = async () => {
+  try {
+    const quoted = await fetch("https://api.quotable.io/quotes?page=1");
+    const parsedQuote = await quoted.json();
+    quotes = parsedQuote.results;
+  } catch (error) {
+    console.error("Error fetching quotes:", error);
+  }
+};
+fetchQuotes();
 const app = express();
 // Now register handlers for some routes:
 //   /                  - Return some helpful welcome info (text)
 //   /quotes            - Should return all quotes (json)
 //   /quotes/random     - Should return ONE quote (json)
+
 app.get("/", (request, response) => {
   response.send("Neill's Quote Server!  Ask me for /quotes/random, or /quotes");
 });
+app.get("/quotes", (request, response) => {
+  response.send({ quotes });
```

Comment:
This looks like it returns an object containing an array, rather than just an array - both of these work, but I'm curious why you chose one over the other?

Reply:
My intention is to send an object that contains an array with a specific name.

Reply:
Can you talk through why you'd rather return an object here than an array? (I'm not saying this is wrong, just interested to explore the trade-offs!)